Project2/GuardedCommands/Script.fsx

```diff
@@ -27,11 +27,17 @@ open CompilerUtil
 
 open Machine
 open VirtualMachine
-
+open Testing
 
 
 System.IO.Directory.SetCurrentDirectory __SOURCE_DIRECTORY__;;
 
+// testing the test module
+
+let fail = Testing.test [0;0] ["gcs-errors/Ex0.gc"; "gcs-errors/Ex1.gc"]
+let pass = Testing.test [0;0;0;0;0;0;0] ["gcs/Ex1.gc"; "gcs/Ex2.gc";"gcs/Ex3.gc"; "gcs/Ex4.gc"; "gcs/Ex5.gc"; "gcs/Ex6.gc"; "gcs/Skip.gc"]
```

Comment:
Is it possible to combine these into tuples? It can quickly become hard to keep track of what entity corresponds to what in the other list.

So like [(0, "gcs/Ex1.gc"); (0, "gcs/Ex2.gc")]

Reply:
that sounds more intuitive, i'll change it to that